docs/firefly-api-overview.md

```diff
@@ -0,0 +1,551 @@
+| use | string | optional, one of 'catalog_overlay', 'catalog_primary', 'data_primary' |
+| tbl_id | string | a unique id of a table, auto-created if not given |
+
+- `firefly.showTable(targetDiv, request, options)`
+
+Parameters ia an object literal.  div is the div to load the table into.
+Below is a list of all possible parameters.
+
+
+| Parameter  | Type | Description |
+| ---------- | ---- | ----------- |
+| targetDiv | string | target div to put the table in |
+| request  | object | request object the table is created from |
+| options  |number | table options |
+
+options -  table options, object with the following properties:
+
+
+| Property  | Type | Description |
+| ---------- | ---- | ----------- |
+| tbl_group | string | the group this table belongs to, defaults to 'main' |
+| removable  | boolean | true if this table can be removed from view, defaults to true |
+| showUnits  | boolean | defaults to false |
+| showFilters | boolean | defaults to false |
+| selectable | boolean | defaults to true |
+| expandable | boolean | defaults to true |
+| showToolbar | boolean | defaults to true |
+| border | boolean | defaults to true |
+
+
+*Example:*
+```js
+tblReq = firefly.util.table.makeFileRequest(null, 'http://web.ipac.caltech.edu/staff/roby/demo/WiseDemoTable.tbl',null,
+            { 
+                pageSize: 15,
+                META_INFO: {CENTER_COLUMN: 'crval1;crval2;EQJ2000'}
+            });
+firefly.showTable('table_div', tblReq);
+```
+The Table tools currently supports the following file formats:
+
+ - IPAC Table file format
+ - CSV - first row should be column headers
+ - TSV - first row should be column header
+ - FITS Tables
+
+
+
+###XY Plot Visualization
+
+- `firefly.showPlot(targetDiv, parameters)`
+
+| Parameter  | Type | Description |
+| ---------- | ---- | ----------- |
+| targetDiv | string | target div to put the XY Plot in |
+| parameters  | object | XY plot parameters |
+
+
+Parameters object literal can contain the following values
+
+| Parameter | Description |
+| ------------ | ------------- |
+| source | required; location of the ipac table.  url or file path.|
+| xCol | column or expression to use for x values, can contain multiple column names ex. log(col) or (col1-col2)/col3 |
+| yCol | column or expression to use for y values, can contain multiple column names ex. log(col) or (col1-col2)/col3 |
+
+
+*Example:*
+```js
+chartParams = {
+    source: 'http://web.ipac.caltech.edu/staff/roby/demo/WiseDemoTable.tbl',
+    xCol: 'ra1',
+    yCol: 'dec1'
+};
+firefly.showPlot('chart_div', chartParams);
+```
+
+XY Plot supports the same table formats as Table does:
+
+ - IPAC Table file format
+ - CSV - first row should be column headers
+ - TSV - first row should be column headers
+ - FITS Tables
+
+
+###Connecting Coverage plot to table
+
+`firefly.showCoverage= (div,options)` - add a coverage plot to a div
```

Comment:
change coverage plot to coverage image. 
it is confusing since showPlo()t is display XY plot.